config/controller.yaml

```diff
@@ -22,6 +22,10 @@ kind: ConfigMap
 metadata:
   name:  lifecycle-image
   namespace: kpack
+  annotations:
+    lifecycle.kpack.io/platformApiVersions: "0.3 0.4"
```

Comment:
is there a reason why these are space separated and not comma seperated

Reply:
No opinion about this. I guess it's whatever's clearer between:
`lifecycle.kpack.io/platformApiVersions: "0.3 0.4"`
`lifecycle.kpack.io/platformApiVersions: "0.3,0.4"`

Reply:
comma is clearer.